packages/core/src/interfaces/Config.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Defines the SDK configuration that can be passed into `initialize`
+ */
+export interface SDKConfig {
+  ios?: {
+    appId: string;
+    appGroupId?: string;
+    disableCrashReporter?: boolean;
```

Comment:
mmm shouldn't be there other attribute to set other crash reporter than us? like crashlitics or something like that, or that is managed in other place

Reply:
this was to keep feature parity for what we currently talk about in the docs: https://embrace.io/docs/react-native/integration/add-embrace-sdk/

so allowing the user to just disable our crash reporter. Automatically configuring crashlytics in place of ours would be more complicated and something we decided not to provide for now, a customer would have to do this themselves in objective-c / swift